app/frontend/stylesheets/config/_variables.scss

```diff
@@ -1,14 +1,11 @@
 @import url('https://fonts.googleapis.com/css2?family=Cinzel+Decorative:wght@700&family=Comfortaa:wght@300&family=Poppins:wght@100;400;500&display=swap');
-
 // Borders & Box Shadows
-
 // Colours
+
 $blue: #7f8ea8;
-$green: #85b6aa;
 
 // Spacing
 $padding: 0.5rem 1rem;
-
 // Fonts'Comfortaa', cursive;
 $body-font: 'Comfortaa', cursive;
 $headers-font: 'Poppins', sans-serif;
```

Comment:
- [ ] There should be an empty space after the import
- [ ] There should be an empty space after // Border
- [ ] The should not be an empty space after // Colours
- [ ] You should not have deleted the green
- [ ] There should be an empty space after $padding